diff --git a/python/scripts/casapy.py b/python/scripts/casapy.py
index 1ab114c..16eeb92 100644
--- a/python/scripts/casapy.py
+++ b/python/scripts/casapy.py
@@ -149,12 +149,12 @@
          },
          'dirs': {
              'rc': homedir + '/.casa',
-             'data': None,
+             'data': "@CASADATAROOT@/share/casa-data/",
              'recipes': None,
-             'root': None,
-             'python': None,
+             'root': '@CASAGCWRAPROOT@',
+             'python':  '@CASAGCWRAPROOT@/python/2.7/',
              'pipeline': None,
-             'xml': None
+             'xml': '@CASAGCWRAPROOT@/xml/'
          },
          'flags': { },
          'files': { 
@@ -170,59 +170,9 @@
 ## ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
 ## set up casa root
 ## ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
-if os.environ.has_key('CASAPATH') :
-    __casapath__ = os.environ['CASAPATH'].split(' ')[0]
-    __casaarch__ = os.environ['CASAPATH'].split(' ')[1]
-    if not os.path.exists(__casapath__ + "/data") :
-        print "DEBUG: CASAPATH = %s" % (__casapath__)
-        raise RuntimeError, "Unable to find the data repository directory in your CASAPATH. Please fix."
-    else :
-        casa['dirs']['root'] = __casapath__
-        casa['dirs']['data'] = __casapath__ + "/data"
-        if os.path.exists(__casapath__ + "/" + __casaarch__ + "/python/2.7/assignmentFilter.py"):
-            casa['dirs']['python'] = __casapath__ + "/" + __casaarch__ + "/python/2.7"
-        elif os.path.exists(__casapath__ + "/lib/python2.7/assignmentFilter.py"):
-            casa['dirs']['python'] = __casapath__ + "/lib/python2.7"
-        elif os.path.exists(__casapath__ + "/Resources/python/assignmentFilter.py"):
-            casa['dirs']['python'] = __casapath__ + "/Resources/python"
-
-        if casa['dirs']['python'] is not None:
-            casa['dirs']['recipes'] = casa['dirs']['python'] + "/recipes"
-
-        if os.path.exists(__casapath__ + "/" + __casaarch__ + "/xml"):
-            casa['dirs']['xml'] = __casapath__ + "/" + __casaarch__ + "/xml"
-        elif os.path.exists(__casapath__ + "/xml"):
-            casa['dirs']['xml'] = __casapath__ + "/xml"
-        else:
-            raise RuntimeError, "Unable to find the XML constraints directory in your CASAPATH"
-else :
-    __casapath__ = casac.__file__
-    while __casapath__ and __casapath__ != "/" :
-        if os.path.exists( __casapath__ + "/data") :
-            break
-        __casapath__ = os.path.dirname(__casapath__)
-    if not os.path.exists(__casapath__ + "/data") :
-        raise RuntimeError, "casa path could not be determined"
-    else :
-        casa['dirs']['root'] = __casapath__
-        casa['dirs']['data'] = __casapath__ + "/data"
-        if os.path.exists(__casapath__ + "/" + __casaarch__ + "python/2.7/assignmentFilter.py"):
-            casa['dirs']['python'] = __casapath__ + "/" + __casaarch__ + "/python/2.7"
-        elif os.path.exists(__casapath__ + "/lib/python2.7/assignmentFilter.py"):
-            casa['dirs']['python'] = __casapath__ + "/lib/python2.7"
-        elif os.path.exists(__casapath__ + "/Resources/python/assignmentFilter.py"):
-            casa['dirs']['python'] = __casapath__ + "/Resources/python"
-
-        if casa['dirs']['python'] is not None:
-            casa['dirs']['recipes'] = casa['dirs']['python'] + "/recipes"
-
-        if os.path.exists(__casapath__ + "/" + __casaarch__ + "/xml"):
-            casa['dirs']['xml'] = __casapath__ + "/" + __casaarch__ + "/xml"
-        elif os.path.exists(__casapath__ + "/xml"):
-            casa['dirs']['xml'] = __casapath__ + "/xml"
-        else:
-            raise RuntimeError, "Unable to find the XML constraints directory in your CASAPATH"
 
+__casapath__ = "/"
+__casaarch__ = "linux"
 
 ## ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
 ## setup pipeline path (if it exists)
@@ -774,22 +724,13 @@
 	if(has_othertasks) :
 	   has_task = myf['task_location'].has_key(myf['taskname'])
 	   if (has_task) :
-		pathname=myf['task_location'][myf['taskname']]
+               pathname=myf['task_location'][myf['taskname']]
 	   else :
-	        pathname = os.environ.get('CASAPATH').split()[0]+'/'+os.environ.get('CASAPATH').split()[1]+'/xml'
-                if not os.path.exists(pathname) :
-                    pathname = os.environ.get('CASAPATH').split()[0]+'/xml'
-                    if not os.path.exists(pathname) :
-                        pathname = os.environ.get('CASAPATH').split()[0]+'/Resources/xml'
-                
+               pathname = casa['dirs']['xml']
 	else :
-	   pathname = os.environ.get('CASAPATH').split()[0]+'/'+os.environ.get('CASAPATH').split()[1]+'/xml'
-           if not os.path.exists(pathname) :
-               pathname = os.environ.get('CASAPATH').split()[0]+'/xml'
-               if not os.path.exists(pathname) :
-                   pathname = os.environ.get('CASAPATH').split()[0]+'/Resources/xml'
-                   if not os.path.exists(pathname) :
-                       sys.exit("ERROR: casapy.py update_params() can not locate xml file for task %s" % (taskname))
+            pathname = casa['dirs']['xml']
+            if not os.path.exists(pathname):
+                sys.exit("ERROR: casapy.py update_params() can not locate xml file for task %s" % (taskname))
                        
         xmlfile=pathname+'/'+myf['taskname']+'.xml'
         if(os.path.exists(xmlfile)) :
