diff --git a/code/CMakeLists.txt b/code/CMakeLists.txt
index e9a1432..05c2a03 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1102,9 +1102,12 @@ if ( ${dbusxx} STREQUAL dbus-c++ )
   # Debian arch specific header location (64 bit only)
   list( APPEND dbus_prefix /usr/lib/x86_64-linux-gnu/dbus-1.0)
 
+  find_package(PkgConfig)
+  pkg_check_modules(PC_DBUS QUIET dbus-1)
+
   casa_find( DBUSXX_ARCH
     INCLUDES dbus/dbus-arch-deps.h
-    PREFIX_HINTS /usr/lib64/dbus-1.0 ${DBUS_ARCH_DIR} ${dbus_prefix}
+    PREFIX_HINTS /usr/lib64/dbus-1.0 ${DBUS_ARCH_DIR} ${dbus_prefix} ${PC_DBUS_INCLUDE_DIRS}/../
     NO_CHECK   # This header does not compile if included directly
     NO_REQUIRE
     IGNORE /usr/lib64/casapy
diff --git a/casaqt/CMakeLists.txt b/casaqt/CMakeLists.txt
index 111a277..44fd922 100644
--- a/casaqt/CMakeLists.txt
+++ b/casaqt/CMakeLists.txt
@@ -220,6 +220,12 @@ casa_add_library( casaqt
   QwtPlotter/QPShape.cc
   QwtPlotter/QPTool.cc
   )
+
+if (UseCrashReporter)
+   # Note the prefix with lib introduced by casa_add_library
+   target_link_libraries (libcasaqt general ${PC_BREAKPAD_LIBRARIES} ${PC_BREAKPAD_CLIENT_LIBRARIES})
+endif ()
+
 install (FILES
 	PlotterImplementations/PlotterImplementations.h
 	DESTINATION include/casacode/casaqt/QwtPlotter
diff --git a/install/target.cmake b/install/target.cmake
index 343a12a..27045db 100644
--- a/install/target.cmake
+++ b/install/target.cmake
@@ -593,7 +593,7 @@ macro (casa_add_google_test)
                             LIBRARIES ${libraries} # gtest + provide libs
                             INCLUDE_DIRS ${gtestIncludeDirectory}) # gtest include dirs
 
-        add_dependencies (${CasaTestName} ${GoogleTest_Target})
+#    add_dependencies (${CasaTestName} ${GoogleTest_Target})
 
     endif ()
 
@@ -641,4 +641,4 @@ macro (casa_add_demo module)
 
   add_dependencies (${executableName} lib${module})
 
-endmacro ()
\ No newline at end of file
+endmacro ()
