diff --git a/CMakeLists.txt b/CMakeLists.txt
index 76ed5f5..6ba0adf 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1075,9 +1075,12 @@ if ( ${dbusxx} STREQUAL dbus-c++ )
   # Debian arch specific header location (64 bit only)
   list( APPEND dbus_prefix /usr/lib/x86_64-linux-gnu/dbus-1.0)
 
+  find_package(PkgConfig)
+  pkg_check_modules(PC_DBUS QUIET dbus-1)
+
   casa_find( DBUSXX_ARCH
     INCLUDES dbus/dbus-arch-deps.h
-    PREFIX_HINTS /usr/lib64/dbus-1.0 ${DBUS_ARCH_DIR} ${dbus_prefix}
+    PREFIX_HINTS /usr/lib64/dbus-1.0 ${DBUS_ARCH_DIR} ${dbus_prefix} ${PC_DBUS_INCLUDE_DIRS}/../
     NO_CHECK   # This header does not compile if included directly
     NO_REQUIRE
     IGNORE /usr/lib64/casapy
@@ -1308,7 +1311,7 @@ casa_find( LIBSAKURA
 #                         #
 ###########################
 
-include (GoogleTestInstall)
+
 
 #########################
 #                       #
@@ -1316,7 +1319,9 @@ include (GoogleTestInstall)
 #                       #
 #########################
 
-include (BreakpadInstall)
+find_package(PkgConfig)
+pkg_check_modules(PC_BREAKPAD QUIET breakpad)
+
 
 ###
 
diff --git a/crash/CMakeLists.txt b/crash/CMakeLists.txt
index 1e8b4b5..b0bdd73 100644
--- a/crash/CMakeLists.txt
+++ b/crash/CMakeLists.txt
@@ -1,5 +1,5 @@
 if (UseCrashReporter)
-    casa_add_executable (NONE CrashReportPoster apps/reporter/CrashReportPoster.cc ${Breakpad_IncludeRoot}/common/linux/http_upload.cc)
+    casa_add_executable (NONE CrashReportPoster apps/reporter/CrashReportPoster.cc ${PC_BREAKPAD_INCLUDEDIR}/common/linux/http_upload.cc)
     target_link_libraries (CrashReportPoster -ldl)
-    target_include_directories (CrashReportPoster PUBLIC ${Breakpad_IncludeRoot})
+    target_include_directories (CrashReportPoster PUBLIC ${PC_BREAKPAD_INCLUDEDIR})
 endif ()
diff --git a/install/target.cmake b/install/target.cmake
index 343a12a..27045db 100644
--- a/install/target.cmake
+++ b/install/target.cmake
@@ -593,7 +593,7 @@ macro (casa_add_google_test)
                             LIBRARIES ${libraries} # gtest + provide libs
                             INCLUDE_DIRS ${gtestIncludeDirectory}) # gtest include dirs
 
-        add_dependencies (${CasaTestName} ${GoogleTest_Target})
+#    add_dependencies (${CasaTestName} ${GoogleTest_Target})
 
     endif ()
 
@@ -641,4 +641,4 @@ macro (casa_add_demo module)
 
   add_dependencies (${executableName} lib${module})
 
-endmacro ()
\ No newline at end of file
+endmacro ()
diff --git a/stdcasa/CMakeLists.txt b/stdcasa/CMakeLists.txt
index 2a53142..ed26584 100644
--- a/stdcasa/CMakeLists.txt
+++ b/stdcasa/CMakeLists.txt
@@ -28,8 +28,9 @@ casa_add_library( stdcasa
 	${CMAKE_CURRENT_BINARY_DIR}/version.cc
 	)
 if (UseCrashReporter)
-    target_include_directories (libstdcasa PUBLIC ${Breakpad_IncludeRoot})
-    target_link_libraries (libstdcasa general ${Breakpad_Library})
+    target_include_directories (libstdcasa PUBLIC ${PC_BREAKPAD_INCLUDE_DIRS})
+    target_link_libraries (libstdcasa general ${PC_BREAKPAD_LIBRARIES})
+    target_compile_options(libstdcasa PUBLIC ${PC_BREAKPAD_CFLAGS_OTHER})    
 endif ()
 
 if (UseCrashReporter)
